modules/OFConnectionManager2/module/inc/OFConnectionManager/ofconnectionmanager.h

```diff
@@ -0,0 +1,119 @@
+/****************************************************************
+ *
+ *        Copyright 2013, Big Switch Networks, Inc.
+ *
+ * Licensed under the Eclipse Public License, Version 1.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *        http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the
+ * License.
+ *
+ ****************************************************************/
+
+/**
+ * @file
+ *
+ * @addtogroup OFConnectionManager
+ * @{
+ *
+ * This file defines the implementation-specific interfaces exposed by
+ * OFConnectionManager. Most functions provided by this module are defined
+ * in the Indigo @ref indigo-OFConnectionManager headers.
+ */
+
+
+#ifndef __OFCONNECTIONMANAGER_H__
+#define __OFCONNECTIONMANAGER_H__
+
+#include "ofconnectionmanager_config.h"
+
+#include <indigo/error.h>
+
+#include <OFConnectionManager/ofconnectionmanager_config.h>
+
+#include <indigo/of_connection_manager.h>
+#include <indigo/of_state_manager.h>
+#include <SocketManager/socketmanager.h>
+
+/**
+ * Global (read-only) counter for connection related errors.
+ */
+
+extern uint32_t ind_cxn_internal_errors;
```

Comment:
this counter has been removed, but it's still in the header file to minimize changes in that file.  i'll remove it when i integrate with pswitch.